docs/decisions/0006-make-redux-deps-optional.rst

```diff
@@ -0,0 +1,51 @@
+Consumers of frontend-platform that don't use redux are still required to install redux dependencies
+==========================================
+
+Status
+------
+
+Review
+
+Context
+-------
+
+Some consumers of frontend-platform, such as frontend-app-learner-portal-enterprise, no longer use redux or react-redux.
+However, they are still required to install these dependencies due to the usage of react-redux in AppProvider, and the fact that
+react-redux and redux are peerDependencies.
+
+This means there is extra overhead and confusion in such projects, when they are not leveraging the `store` feature of AppProvider.
```

Comment:
It might be good to clarify that `AppProvider` uses the `OptionalReduxProvider` which conditionally uses Redux only when `store` is provided aka the use of a Redux store is optional from a code perspective but not from a dependency perspective.